internal/connect/api.go

```diff
@@ -176,14 +184,45 @@ func updateSystem(body []byte) error {
 	return err
 }
 
+// readUptimeLogFile reads the system uptime log from a given file and
+// returns them as a string array. If the given file does not exist,
+// it will be interpreted as if the system uptime log feature is not
+// enabled. Hence an empty array will be returned.
+func readUptimeLogFile(uptimeLogFilePath string) ([]string, error) {
+	// NOTE: if uptime log file does not exist, we assume the uptime
+	// tracking feature is not enabled
```

Comment:
I'd maybe add a note for reference as to where this file comes from, and that the existence of this file is guaranteed when using "whatever external tool" in "that environment" 😉